pkg/controller/proxyconfig/validation.go

```diff
@@ -34,8 +34,9 @@ func (r *ReconcileProxyConfig) ValidateProxyConfig(proxyConfig *configv1.ProxySp
 		if err != nil {
 			return fmt.Errorf("invalid httpsProxy URI: %v", err)
 		}
-		if scheme != proxyHTTPScheme && scheme != proxyHTTPSScheme {
-			return fmt.Errorf("httpsProxy requires a %q or %s URI scheme", proxyHTTPScheme, proxyHTTPSScheme)
+		// TODO: Add proxyHTTPSScheme when CA support is added.
+		if scheme != proxyHTTPScheme {
+			return fmt.Errorf("httpsProxy requires a %s URI scheme", proxyHTTPScheme)
```

Comment:
i don't see a reason not to allow https today.  it will work for passthrough proxies.

Reply:
@danehans bump

Reply:
@bparees I addressed your comments in my latest commit.